### PR TITLE
feat(notification): discord integration

### PR DESCRIPTION
**Pull request by bmalaski**

### CODE REVIEW

File: src/notification/discord.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+			embed.setTitle('Stock Notification');
+			embed.addField('URL', text, true);
+			embed.addField('Brand', link.brand, true);
+			embed.addField('Model', link.model, true);
+
+			if (notifyGroup !== '') {
+				embed.addField('Attention', notifyGroup, true);
+			}
+
+			embed.setColor(65340);
```
Comment (jef) [inline src/notification/discord.ts:22]: I believe this needs to be a hex string. I like this color :smile: `#58D68D`

File: src/notification/discord.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import {Webhook, MessageBuilder} from 'discord-webhook-node';
+import {Config} from '../config';
+import {Logger} from '../logger';
+import {Link} from '../store/model';
+
+const hook = new Webhook(Config.notifications.discord.webHookUrl);
+const notifyGroup = Config.notifications.discord.notifyGroup;
+
+export function sendDiscordMessage(text: string, link: Link) {
```
Comment (jef) [inline src/notification/discord.ts:9]: Let's change this to `cartUrl` just to be consistent.

```suggestion
export function sendDiscordMessage(cartUrl: string, link: Link) {
```

File: .env-example [hunk 3]
```diff
@@ -18,3 +18,5 @@ STORES="bestbuy,bandh,nvidia"
 SCREENSHOT="true"
 TELEGRAM_ACCESS_TOKEN=""
 TELEGRAM_CHAT_ID="1234"
+DISCORD_WEB_HOOK="https://discordapp.com/api/webhooks/23123123123/5dfsdfh3h2j5hdfhsdfsdhj55jf"
+DISCORD_NOTIFY_GROUP="@here"
```
Comment (jef) [inline .env-example:22]: Do you mind putting this in lexicographical order? Same in the `README.md`?

**Review by jef — CHANGES_REQUESTED** [review]
Just a few things, otherwise, looks great!

File: src/notification/discord.ts (hunk 1 above)
Comment (bmalaski) [reply src/notification/discord.ts:22]: Sure I was just copying what they used to for the webhooks.info color, but I can change it to NVIDIA green #76b900

Comment (jef) [reply src/notification/discord.ts:22]: Even better!

File: src/notification/discord.ts (hunk 2 above)
Comment (bmalaski) [reply src/notification/discord.ts]: sure no problem

File: src/notification/discord.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+			embed.setTitle('Stock Notification');
+			embed.addField('URL', cartUrl, true);
+			embed.addField('Brand', link.brand, true);
+			embed.addField('Model', link.model, true);
+
+			if (notifyGroup !== '') {
+				embed.addField('Attention', notifyGroup, true);
+			}
+
+			embed.setColor(0x76b900);
```
Comment (jef) [inline src/notification/discord.ts:22]: I think you want to use a string here.

Comment (bmalaski) [reply src/notification/discord.ts]: nope just need to use the right case for linter to be happy. This is the correct useage.

**Review by jef — APPROVED** [review]
Looks good! Thanks for your contribution! Whats your Discord name?

**Pull request merged**